src/control/cmd/dmg/server_test.go

```diff
@@ -0,0 +1,43 @@
+//
+// (C) Copyright 2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+import (
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/lib/control"
+	"github.com/pkg/errors"
```

Comment:
nit: imports

Reply:
fixed in client side validation follow-up